<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearancePage</class>
 <widget class="QWidget" name="AppearancePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="appearancePageContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1052</width>
        <height>583</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_1337">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="appearanceFrame">
         <layout class="QVBoxLayout" name="verticalLayout_30">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="appearanceGeneral">
            <property name="title">
             <string>Basic.Settings.Appearance</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout_37">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Basic.Settings.Appearance.General.Theme</string>
               </property>
               <property name="buddy">
                <cstring>theme</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="theme"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Basic.Settings.Appearance.General.Variant</string>
               </property>
               <property name="buddy">
                <cstring>themeVariant</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="themeVariant"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="appearanceSettingLabelFontScale">
               <property name="text">
                <string>Basic.Settings.Appearance.FontScale</string>
               </property>
               <property name="buddy">
                <cstring>appearanceFontScale</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="0,5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="appearanceFontScaleText">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::FocusPolicy::NoFocus</enum>
                  </property>
                  <property name="text">
                   <string>10</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="AbsoluteSlider" name="appearanceFontScale">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimum">
                   <number>8</number>
                  </property>
                  <property name="maximum">
                   <number>12</number>
                  </property>
                  <property name="pageStep">
                   <number>2</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::TickPosition::TicksBothSides</enum>
                  </property>
                  <property name="tickInterval">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Basic.Settings.Appearance.Density</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_34">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="appearanceDensity1">
                  <property name="text">
                   <string>Basic.Settings.Appearance.Classic</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">appearanceDensityButtonGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="appearanceDensity2">
                  <property name="text">
                   <string>Basic.Settings.Appearance.Compact</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">appearanceDensityButtonGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="appearanceDensity3">
                  <property name="text">
                   <string>Basic.Settings.Appearance.Normal</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">appearanceDensityButtonGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="appearanceDensity4">
                  <property name="text">
                   <string>Basic.Settings.Appearance.Comfortable</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">appearanceDensityButtonGroup</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="4" column="0">
              <spacer name="horizontalSpacer_17">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>170</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="appearanceOptionsWarning">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_35">
             <item>
              <widget class="QLabel" name="appearanceOptionsWarningLabel">
               <property name="text">
                <string>Basic.Settings.Appearance.OptionsWarning</string>
               </property>
               <property name="class" stdset="0">
                <string>text-warning</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AbsoluteSlider</class>
   <extends>QSlider</extends>
   <header>components/AbsoluteSlider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="appearanceDensityButtonGroup"/>
 </buttongroups>
</ui>
